game/entity/entity.py

```diff
@@ -0,0 +1,17 @@
+from abc import abstractmethod
+
+
+class Entity:
+    def __init__(self, name: str):
+        self._type: str = name
+
+    @abstractmethod
+    def draw(self, screen, font_size) -> None:
```

Comment:
missing annotation (and in other entities)

Reply:
I now added the inheritance of the ABC class to the entity, tank and feature classes, in order to fully declare them as abstract. Is this what you meant by missing annotation or is it something else I am missing?

Reply:
type annotation.
sometimes present and sometimes not